starfish/test/image/imagestack_test_utils.py

```diff
@@ -38,20 +38,21 @@ def verify_stack_fill(
 
 def verify_physical_coordinates(
         stack: ImageStack,
-        selectors: Mapping[Axes, int],
         expected_x_coordinates: Tuple[float, float],
         expected_y_coordinates: Tuple[float, float],
         expected_z_coordinates: Tuple[float, float]
 ) -> None:
-    """Given an imagestack and a set of selectors, verify that the physical coordinates for the data
-    referred to by the selectors match the expected physical coordinates.
+    """Given an imagestack and a set coordinate min/max values
```

Comment:
```suggestion
    """Given an imagestack and a set of coordinate min/max values
```

you can also unwrap a bit, as the line length limit is like 100.